gradle/versions.gradle

```diff
@@ -45,7 +45,7 @@ dependencyManagement {
       entry 'cava-junit'
       entry 'cava-kv'
       entry 'cava-rlpx'
-      entry 'cava-ssz'
+      // entry 'cava-ssz'
```

Comment:
Would you mind adding a comment explaining why this is commented out?

Reply:
19b9e56